Wrong arch type of Zig on macOS server

I just tried to build [example-languages](https://github.com/defold/example-languages) project for x86_64-macos, but it failed with this (`Bad CPU type in executable`):

```
com.defold.extender.ExtenderException: java.io.IOException: Cannot run program "/usr/local/extender-production/platformsdk/zig-0-11/zig" (in directory "/var/folders/2b/mpxyl1m54jvg2rcs2cnbsqxc0000gn/T/job2331304605428515312"): error=86, Bad CPU type in executable
	at com.defold.extender.Extender.buildEngine(Extender.java:2219)
	at com.defold.extender.Extender.build(Extender.java:2587)
	at com.defold.extender.AsyncBuilder.asyncBuildEngine(AsyncBuilder.java:116)
	at jdk.internal.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:354)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:768)
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:113)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.io.IOException: Cannot run program "/usr/local/extender-production/platformsdk/zig-0-11/zig" (in directory "/var/folders/2b/mpxyl1m54jvg2rcs2cnbsqxc0000gn/T/job2331304605428515312"): error=86, Bad CPU type in executable
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1143)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at com.defold.extender.ProcessExecutor.execute(ProcessExecutor.java:43)
	at com.defold.extender.ProcessExecutor$1.call(ProcessExecutor.java:125)
	at com.defold.extender.ProcessExecutor$1.call(ProcessExecutor.java:122)
	... 4 more
Caused by: java.io.IOException: error=86, Bad CPU type in executable
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:314)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:244)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1110)
	... 8 more
```

--- a/server/scripts/standalone/setup-standalone-env.sh
+++ b/server/scripts/standalone/setup-standalone-env.sh
@@ -180,9 +180,11 @@ PACKAGES=(
     XcodeDefault15.4.xctoolchain.darwin
 )
 
-# ZIG_VERSION=0.11.0
-# ZIG_PATH_0_11=${PLATFORMSDK_DIR}/zig-${ZIG_VERSION}
-ZIG_PACKAGE_NAME=zig-macos-x86_64-${ZIG_VERSION}.tar.xz
+ZIG_ARCH=x86_64
+if [[ $(uname -m) == "arm64" ]]; then
+    ZIG_ARCH=aarch64
+fi
+ZIG_PACKAGE_NAME=zig-macos-${ZIG_ARCH}-${ZIG_VERSION}.tar.xz
 ZIG_URL=https://ziglang.org/download/${ZIG_VERSION}
 
 function download_packages() {

